src/stores/cluster/index.js

```diff
@@ -349,115 +265,6 @@ export default class ClusterStore extends Store {
     this.clusterIds = [];
   };
 
-  @action
-  onSearchNode = async word => {
-    this.searchNode = word;
-    this.currentNodePage = 1;
-    await this.fetchNodes({ cluster_id: this.cluster.cluster_id });
-  };
-
-  @action
-  onClearNode = async () => {
-    await this.onSearchNode('');
-  };
-
-  @action
-  onRefreshNode = async () => {
-    await this.fetchNodes({ cluster_id: this.cluster.cluster_id });
-  };
-
-  @action
-  changePaginationNode = async page => {
-    this.currentNodePage = page;
-    await this.fetchNodes({ cluster_id: this.cluster.cluster_id });
-  };
-
-  @action
-  onChangeNodeStatus = async status => {
-    this.currentNodePage = 1;
-    this.selectNodeStatus = this.selectNodeStatus === status ? '' : status;
-    await this.fetchNodes({
-      cluster_id: this.cluster.cluster_id
-    });
-  };
-
-  @action
-  loadNodeInit = () => {
-    this.currentNodePage = 1;
-    this.selectNodeStatus = '';
-    this.searchNode = '';
-  };
-
```

Comment:
Why delete these methods? The cluster detail page has node table, these methods will be used here.